comp/checks/agentcrashdetect/agentcrashdetectimpl/agentcrashdetect.go

```diff
@@ -43,14 +43,16 @@ const (
 var (
 	// crashdriver included for testing purposes
 	ddDrivers = map[string]struct{}{
-		"ddnpm":       {},
-		"crashdriver": {},
+		"ddnpm":       {}, // NPM/USM driver, used for network monitoring
+		"ddprocmon":   {}, // process monitoring driver, used for CWS
+		"crashdriver": {}, // this entry exists only for testing purposes.
```

Comment:
If possible, I think it'd be cleaner if we only included this in the list when we are testing internally. Unfortunately, I'm not such if we have a way to either conditionally compile it in / out or if we could perhaps export ddDrivers and then in test code we could add in to the map(?) Something like that - any ideas?